pybind11_stubgen/__init__.py

```diff
@@ -379,20 +390,19 @@ def parse(self):
 
     def to_lines(self):  # type: () -> List[str]
 
-        docstring = self.remove_signatures(self.prop.__doc__)
-        docstring += "\n:type: {rtype}".format(rtype=self.signature.rtype)
+        docstring = self.filter_docstring(self.prop.__doc__)
+        docstring_prop = "\n\n".join([docstring, ":type: {rtype}".format(rtype=self.signature.rtype)])
 
         result = ["@property",
                   "def {field_name}(self) -> {rtype}:".format(field_name=self.name, rtype=self.signature.rtype),
-                  self.indent('"""{}"""'.format(docstring))]
+                  self.indent_lines('"""\n{}\n"""'.format(docstring_prop.strip("\n")))]
```

Comment:
This operations on `\n` seems to be repeated as well and needs to be extracted to a new function too.